synapse/app/frontend_proxy.py

```diff
@@ -135,6 +165,12 @@ def _listen_http(self, listener_config):
                 elif name == "client":
                     resource = JsonResource(self, canonical_json=False)
                     KeyUploadServlet(self).register(resource)
+
+                    # If presence is disabled, use the stub servlet that does
+                    # not allow sending presence
+                    if not self.config.use_presence:
+                        PresenceStatusStubServlet(self).register(resource)
+
```

Comment:
Might be worth mentioning in `docs/workers.rst` that you can point presence endpoints at frontend worker when presence is disabled?

Reply:
done